odoo/tools/template_inheritance.py

```diff
@@ -217,6 +217,28 @@ def extract(spec):
                     if child.get('position') == 'move':
                         child = extract(child)
                     node.addprevious(child)
+            elif pos == 'content':
+                ''' Replace the entire content of an element '''
+                target = copy.deepcopy(node)
+                # Clear the content and attributes
+                node.clear()
+                # Get back attributes
+                for attr in target.keys():
+                    node.set(attr, target.get(attr))
+
+                # Add xpath content
+                if spec.text is not None:
+                    node.text = spec.text.lstrip()
+                for child in spec.getchildren():
+                    node.append(child)
+
+                # If the new content has children, right strip the tail of the last one
+                childs = node.getchildren()
```

Comment:
childs is not an english work... and you are calling `getchildren` just after :smile: That method is deprecated according to the doc, so try not to use it.